src/platform/BUILD.gn

```diff
@@ -54,7 +54,7 @@ if (chip_device_platform != "none") {
     chip_bypass_rendezvous = false
 
     # Enable including the additional data in the advertisement packets
-    chip_enable_additional_data_advertising = true
+    chip_enable_additional_data_advertising = false
```

Comment:
Please confirm that it does not affect any other platforms

Reply:
Added `chip_enable_additional_data_advertising = true` to tv-app and tv-cast-app.